unscrobbler.js

```diff
@@ -70,6 +70,12 @@ scrobbleRows.forEach(row => {
 
   // Insert the checkbox before the track heart icon.
   row.insertBefore(checkboxTableData, loveTableData);
+
+  row.addEventListener('click', (e) => {
```

Comment:
Just to match with the previous `event` code occurences (e.g. in line 18)
```suggestion
  row.addEventListener('click', (event) => {
```